safe/impact_template/road_building_report_template.py

```diff
@@ -41,14 +41,80 @@ def __init__(
         :param impact_data: Dictionary that represent impact data.
         :type impact_data: dict
         """
-        super(AbstractRoadBuildingReportTemplate, self).__init__(
+        super(RoadBuildingReportTemplate, self).__init__(
```

Comment:
What happens if we need another exposure type (Foo) in the future? Will we make it RoadBuildingFooReportTemplate ?